openvinotoolkit/cvat
# Annotation filters UI

### Motivation and context
[1418: CVAT new UI: find the way to simplify the syntax for filters](https://github.com/openvinotoolkit/cvat/issues/1418)

### How has this been tested?

- Go to annotation page
- Click to filters input
- Play around with form
- **Shift+Left click: add open bracket for item**
- **Shift+Right click: add close bracket for item**
- **Alt+Left click: remove open bracket for item**
- **Alt+Right click: remove close bracket for item**
- **Right click on filter input open confirm dialog to clean up filters**

### Checklist

  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```
